prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2342,17 +2293,16 @@ int rxr_endpoint(struct fid_domain *domain, struct fi_info *info,
 	if (ret)
 		goto err_close_ofi_ep;
 
-	ret = efa_ep_open(&efa_domain->util_domain.domain_fid, rdm_info,
-			  &rxr_ep->rdm_ep, rxr_ep);
+	ret = efa_base_ep_construct(&rxr_ep->base_ep, rdm_info);
```

Comment:
line 2282 to 2283, the call to `ofi_endpoint_init` to initialize `base_ep.util_ep` should be merged into `efa_base_ep_construct`

Reply:
Need to pass the context, util_prov and the progress function as a function pointer to `efa_base_ep_construct`. I didn't think that was worth the complexity.

Reply:
Please add a todo note here that we will revist once we merge `rxr_prov_util` and `efa_prov_util`

Reply:
done